### PR TITLE
Add Cloudflare DNS config option

### PR DESCRIPTION
**Pull request by swrobel**
**Status**: Ready

**Reference**: https://github.com/Fourdee/DietPi/issues/2094#issuecomment-429525510

### CODE REVIEW

File: dietpi/dietpi-config [hunk 1]
```diff
@@ -2744,6 +2744,7 @@ _EOF_
 			'37.235.1.174 37.235.1.177' ': FreeDNS'
 			'64.6.64.6 64.6.65.6' ': Verisign'
 			'84.200.69.80 84.200.70.40' ': DNS Watch'
+			'1.1.1.1 1.0.0.1': 'Cloudflare'
```
Comment (Fourdee) [inline dietpi/dietpi-config]: ```:``` appears to be in the wrong location, needs to be inside the 2nd array string:
eg:
```
'1.1.1.1 1.0.0.1' ': Cloudflare'
```

**Review by Fourdee — CHANGES_REQUESTED** [review]
Many thanks for your PR 👍 

Please see the 1 required change before we can commit.

**Comment by swrobel** [comment]
My mistake. Thanks for taking a careful look and catching that!

**Review by Fourdee — APPROVED** [review]
Perfect, thank you 👍

**Comment by Fourdee** [comment]
@swrobel 

Please email me your address, i'll send you some DietPi stickers for your first PR 👍 :
daniel.knight@dietpi.com

**Pull request merged**